go/consensus/tendermint/apps/roothash/api.go

```diff
@@ -29,21 +29,21 @@ var (
 
 	// KeyRuntimeID is an ABCI event attribute key for specifying event
 	// runtime (value is Base64-encoded runtime ID).
-	KeyRuntimeID = []byte("runtime-id")
+	KeyRuntimeID = []byte("runtime_id")
```

Comment:
Aw, and this was supposed to end up being something that was potentially backwards compatible. :(